build/cloudwatch-prometheus-exporter.spec

```diff
@@ -38,6 +38,8 @@ rm -rf %{buildroot}
 
 
 %changelog
+* Wed Jul 19 2023 Rajashree Pethe <rajashree@covergenius.com>
+- Adding metrics for elasticche 
```

Comment:
As we might add more metrics in a future lets specify which ones were added as list is short